src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,86 @@
+package task;
+
+import file.FileManager;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+    private static int taskCount = 0;
+    private static final String LINE_DIVIDER = "-------------------------------------";
+    private static final String ROGER_MESSAGE = "Got it. I have added this task:";
+
+    private static void printRoger(Task task) {
+        System.out.println(ROGER_MESSAGE);
+        System.out.println(task);
+        System.out.println("You now have " + taskCount + " tasks.");
+        System.out.println(LINE_DIVIDER);
+    }
+
+    private static void printMark(int target, boolean mark) {
```

Comment:
Consider renaming the `mark` variable to `isMarked`, to keep consistency in naming convention.